lab-geoff/route/routes.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let jsonParser = require('body-parser').json();
+let User = require('../model/model.js');
+let auth = require('../lib/auth.js');
+let Router = require('express').Router;
+let router = new Router();
+
+router.post('/users', jsonParser, (req, res) => {
+  console.log('/users');
+  console.log(req.body);
+  let body = req.body;
+  console.log(body);
+  let user = new User(body);
+  user.hashPass(user.password)
+    .then(user => user.save())
+    .then(user => res.json(user))
+    .catch();
+});
+
+router.get('/users/:id', auth, function(req, res) {
+  console.log('/users/:id');
+  User.findById(req.params.id)
+  .then(user => {
```

Comment:
This get route looks solid. If you're having troubles I would suggest circling back to the auth middleware and making sure that things are set correctly there.